www/js/app.js

```diff
@@ -864,7 +869,8 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
                     return;
                 }
                 if (!isServiceWorkerReady()) {
-                    $('#serviceWorkerStatus').html("ServiceWorker API available : trying to register it...");
+                    let serviceWorkerStatus = document.getElementById('serviceWorkerStatus');
```

Comment:
If `serviceWorkerStatus` is still in scope, you can remove this too.

Reply:
Right.

Reply:
I see it's a different function, so not in scope.